Week3/homework/step2-4.js

```diff
@@ -2,6 +2,9 @@
 
 function Dog() {
   // add your code here
+  this.name = 'Max';
+  this.color = 'Grey';
+  this.numLegs = 4;
```

Comment:
You've hard-coded Max, Grey, and 4, into the Dog constructor. That means any hound you create will always have these values.

The idea here is that Dog should take name, color, and numLegs as parameters - which you'd pass in when you create it. Like this;

const hound = new Dog('Max', 'Grey', 4);